spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-configmap/src/test/resources/fabric8-deployment.yaml

```diff
@@ -26,3 +26,6 @@ spec:
               path: /actuator/health/liveness
           ports:
             - containerPort: 8080
```

Comment:
Need to add some integration tests and assert presence of logs, but can be done in a separate PR too, depending on which one is accepted, if any of course